### ISSUE TITLE
Add cryptocurrency prices using bittrex api

### ISSUE THREAD

**Issue by shawnjzlee** [swapagarwal/JARVIS-on-Messenger#282]
Hi There,

First timer here! I'd like to add a functionality that would fetch cryptocurrency ticker prices from the public bittrex api.

e.g. `ticker BTC-LTC` or `market BTC-LTC` would return the real-time tick values for a market.

Could this be in conflict with the existing currency feature? I'm all ears for suggestions!

**Comment by edadesd** [comment]
If you could think of a query that unambiguously calls the cryptocurrency module (like "BTC-LTC cryptocurrency") that would make it easier to direct the query to the appropriate module.

**Comment by shawnjzlee** [comment]
`cryptocurrency`, `crypto`, `ticker`, `market` should call the cryptocurrency module unambiguously (e.g. `cryptocurrency BTC to LTC`, `get ticker for LTC to BTC`)

If I enter the symbols BTC and LTC into the fixer api, the "rates" returns empty. Would it be better to call the bittrex api if the above condition is satisfied?

**Comment by edadesd** [comment]
I think it would be ideal to only call one API per user query. Since you know Fixer won't give you a value for Rates, I would try to make that query rely entirely on Bittrex without calling Fixer. So you would have to implement some logic to decide which API to call based on the user's query.

**Comment by valinsky** [comment]
I was just thinking to add this. Are you working on the feature?

**Comment by shawnjzlee** [comment]
Yep! Feel free to take a look at #283 and make changes as you see fit if you'd like to work on it (:

**Comment by stwins60** [comment]
is this issue still open
